core/src/main/java/com/tweet/core/model/UserRole.java

```diff
@@ -0,0 +1,19 @@
+package com.tweet.core.model;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@Entity
+public class UserRole {
+
+    @Id
```

Comment:
I see that you really enjoy ids a strings ;)
But as we discussed its way much better to make it long.
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private long roleId;